test/e2e/utils/playwright-helpers.ts

```diff
@@ -0,0 +1,14 @@
+import { expect, Locator, Page } from '@playwright/test';
```

Comment:
I am not sure yet whether the name of this file (playwright-helpers.ts) is good.
Do you have other suggestions?

Reply:
seems fine to me for now, if we determine a better name later, we'll change it then.